From 499515921369eabbf6d0278845ef7aaa32e1788e Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:07:07 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit 227e3418ea133ad43f0ee66b816ca143bbd1baa9
Merge: 881b8b7 8d29db3
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jul 1 03:10:43 2018 +0000

    Snap for 4872253 from 8d29db3ee481ced0a4df9831b1bff2642766e85c to pi-dr1-release

    Change-Id: I7fbba18fb383f7fccd53f0654bfa2d044842a12b

commit 8d29db3ee481ced0a4df9831b1bff2642766e85c
Author: Amit Mahajan <amitmahajan@google.com>
Date:   Tue Jun 5 16:31:16 2018 -0700

    Add non-roaming registered PLMN name for Xfinity.

    Test: manual
    Bug: 80145031
    Change-Id: I637b36fd0fe58e1c11100496fedbb52acc67405c

commit 881b8b764a3917c41c932776772ef68e7a21dcb9
Merge: b84e0c2 769c733
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Jun 6 03:01:32 2018 +0000

    Snap for 4823496 from 769c733097b79b5d300d7f613b0c1a20e63a660a to pi-dr1-release

    Change-Id: I8167d4e1a42194fdffe9d8781500581fddb03d83

Change-Id: I96be190904b0f7f63e291d533c07f6ca50223326
---
 assets/carrier_config_311480.xml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/assets/carrier_config_311480.xml b/assets/carrier_config_311480.xml
index d13ff54..7639005 100644
--- a/assets/carrier_config_311480.xml
+++ b/assets/carrier_config_311480.xml
@@ -156,4 +156,9 @@
         <boolean name="carrier_name_override_bool" value="true" />
         <string name="carrier_name_string"></string>
     </carrier_config>
+    <!-- XFINITY Mobile as operator name for Xfinity -->
+    <carrier_config gid1="BA01450000000000">
+        <boolean name="cdma_home_registered_plmn_name_override_bool" value="true" />
+        <string name="cdma_home_registered_plmn_name_string">XFINITY Mobile</string>
+    </carrier_config>
 </carrier_config_list>
-- 
2.17.1

